tests/wpt/metadata/resource-timing/resource_TAO_zero.htm.ini

```diff
@@ -1,22 +1,22 @@
 [resource_TAO_zero.htm]
   [secureConnectionStart should be 0 in cross-origin request.]
-    expected: FAIL
+    expected: PASS
```

Comment:
All of the test files that only contain PASS expectations can now be deleted, since the default test expectation when there is no ini file is PASS.